mobile/src/components/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {View, Image, StyleSheet} from 'react-native';
+
+const Header = () => {
+  return (
+    <View style={styles.container}>
+      <Image
+        source={require('../../assets/ISA-logo.png')}
+        style={styles.logo}
+      />
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: '100%',
+    height: 80,
```

Comment:
I think we can trim down the header by a little bit (e.g., 70-75), its a bit too thick now

Reply:
> Home page is looking super!
> 
> * Let's organize components into logical folder (e.g., per page + shared folder)
> * move vendorMockData.js into src/data folder
> * we can also delete helper/example.jsx and components/Example.jsx if it's no longer needed

Fixed with 07334f2bd1ea6b52d8eb9d25f5c276e298e96c0a, e1fce8ae3eb719d2653acf1688bb6cd8c756eecd and 597e25032d28eb8cd57ea428ddd169a20f9f2c9b